app/models/spreadsheet_service.rb

```diff
@@ -0,0 +1,31 @@
+require 'google/apis/sheets_v4'
+
+class SpreadsheetService
+  def self.get_data(spreadsheet_id)
+    service = Google::Apis::SheetsV4::SheetsService.new
+    service.key = ENV['GOOGLE_SHEET_API_KEY']
```

Comment:
Let's add _some_ documentation regarding this API Key (how to get it, what configurations may be needed).